packages/amplify-app/src/index.js

```diff
@@ -338,6 +339,35 @@ async function createAndroidHelperFiles() {
   }
 }
 
+async function createIosHelperFiles() {
+  const configFile = path.join(process.cwd(), '/amplifyxc.config');
+  const configStr = 'push=false\nmodelgen=false\nprofile=default\nenvName=amplify';
```

Comment:
what type file is this? Ini file?

Reply:
yes its ini. using .config to make it clear what the file is used for

Reply:
Then may be we should use this 
https://www.npmjs.com/package/ini

Reply:
Changed to ini